### DISCUSSION TITLE
Remove Scale

### DISCUSSION THREAD

**Discussion by darumwarum** [sgratzl/chartjs-chart-geo#73] [Q&A]
Hallo,

is there any way to remove the Label? Please have a look at the attached image.

<img width="577" alt="Bildschirmfoto 2021-06-04 um 11 09 58" src="https://user-images.githubusercontent.com/1814905/120777340-78700080-c525-11eb-897b-2b4c090bfebd.png">

Thank you for your help!

**Comment by sgratzl** [comment]
the legend is a regular scale. Thus, you should be able to set `display: false` on the r-scale object. see also https://github.com/sgratzl/chartjs-chart-geo/blob/3d9a21cab2196ee55bc67ba64d14bb898855fab9/samples/bubbleMapLog.html#L54 which points to the place to add it

**Reply by darumwarum** [reply]
Oh that's great. Thank you!

**Reply by james0r** [reply]
adding `display: false` to the r-scale object seems to disable my entire chart.

here's my options block:

```
        options: {
          plugins: {
            legend: {
              display: false
            },
          },
          scales: {
            xy: {
              projection: 'albersUsa'
            },
            r: {
              display: false
            },
            color: {
              quantize: 5,
              legend: {
                position: 'bottom-right',
                align: 'bottom',
              },
            }
          },
        }
```

**Reply by sgratzl** [reply]
can you provide a complete example, please

**Reply by nirgn975** [reply]
@sgratzl  Here it a complete example: https://codepen.io/nirgn975/pen/LYMGgpd

I took the original from the docs(https://codepen.io/sgratzl/pen/YzyJRvm) and just add the `r` scale with `display: false`.
You can see this error on the browser console:
![CleanShot 2023-08-28 at 22 44 45](https://github.com/sgratzl/chartjs-chart-geo/assets/3472902/70c9f270-4f82-4976-b976-f078963940bb)

**Reply by stockiNail** [reply]
Sorry if I'm jumping in the thread.
The "size" scale is an extension of legend scale, therefore to remove the legend, try adding "display: false" to "size" scale.

https://codepen.io/stockinail/pen/YzdwJNR

**Reply by nirgn975** [reply]
@stockiNail thank you, that works!
Adding my `options` for others:
```
options: {
  plugins: { legend: { display: false } },
  scales: {
    projection: { projection: "equalEarth", axis: "x" },
    size: { display: false, axis: "x" },
  },
}
```
